analysis_options.yaml

```diff
@@ -103,7 +103,7 @@ linter:
     - join_return_with_assignment # not yet tested
     - library_names
     - library_prefixes
-    - lines_longer_than_80_chars # not yet tested
+    - lines_longer_than_90_chars # not yet tested
```

Comment:
Is this a new lint rule?

Reply:
My mistake, i changed to test and didnt put back, the correct is 
lines_longer_than_80_chars